pkg/ironic/raid.go

```diff
@@ -165,7 +165,7 @@ func buildTargetHardwareRAIDCfg(volumes []metal3v1alpha1.HardwareRAIDVolume) (lo
 	if len(volumes) == 0 {
 		return
 	}
-
+	fmt.Printf("buildTargetHardwareRAIDCfg volumes: %+v\n\n", volumes)
```

Comment:
Let's not start printing random stuff to stdout without going through the logger.

Reply:
Removed this

Reply:
I have resolved the issues..Please review @zaneb